tests/flow/array-filter/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -41,13 +41,15 @@ console.log(filteredItems);
 /* @flow */
 
 function filterItems(items: Array<string | number>): Array<string | number> {
-  return items.map(item => {
-    if (typeof item === \"string\") {
-      return item.length > 2 ? item : null;
-    } else {
-      return item * 10;
-    }
-  }).filter(Boolean);
+  return items
+    .map(item => {
+      if (typeof item === \"string\") {
+        return item.length > 2 ? item : null;
+      } else {
+        return item * 10;
+      }
+    })
+    .filter(Boolean);
```

Comment:
It feels weird to me that things are not properly indented and flow weirdly. For this one it looks okay because Boolean is small but if it was a function, it wouldn't indent. I feel like it's more consistent to always indent it when you break.

For the example you have in your comment, I replied below that I can special case it if the last function call has just a function as argument.